projects/prdt/index.js

```diff
@@ -14,15 +14,16 @@ const config = {
       predictionBTCPOLY: "0xd71b0366CD2f2E90dd1F80A1F0EA540F73Ac0EF6",
       predictionTESLA: "0x3fc376530Ac35d37Dd1Fa794F922e0f30CbB2c46",
       predictionMATIC: "0x59e0aD27d0F58A15128051cAA1D2917aA71AB864",
+      predictionPRO: "0x764C3Ea13e7457261E5C1AaD597F281f3e738240",
```

Comment:
this does not look like a smart contract: https://www.bscscan.com/address/0x764C3Ea13e7457261E5C1AaD597F281f3e738240

Reply:
my bad, was looking the wrong chain